src/app/utils/utils.service.rUtils.js

```diff
@@ -0,0 +1,118 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('roboconf.utils')
+        .service('rUtils', rUtils);
+
+    rUtils.$inject = [ '$timeout' ];
+    
+    /* @ngInject */
+    function rUtils( $timeout ) {
+        var service = {
+            endsWith: endsWith,
+            startsWith: startsWith,
+            isObj: isObj,
+            sortInstances: sortInstances,
+            findPosition: findPosition,
+            showRightBlock: showRightBlock,
+            hideRightBlock: hideRightBlock
+        };
+
+        return service;
+        /////////////////////
+
+        function startsWith( string, prefix ) {
+        	return isObj( string ) && isObj( prefix ) && string.indexOf( prefix ) === 0;
+        }
+
+        function endsWith( string, suffix ) {
+        	return isObj( string ) && isObj( suffix ) && string.indexOf( suffix, string.length - suffix.length ) !== -1;
+        }
+        
+        function isObj( obj ) {
+        	return obj !== undefined && obj !== null; 
```

Comment:
Bad function name. When seeing that name in javascript we expect that obj is an object ( {} or new Object() )

https://docs.angularjs.org/api/ng/function/angular.isObject